build/bazel/remote/asset/v1/qualifiers.md

```diff
@@ -5,7 +5,27 @@ This lexicon defines standard qualifier names that servers
 
 The following standard qualifier `name`s are defined:
 
-* `resource_type`: This describes the type of resource.
+* `resource_type`: This describes the type of resource.  In case of file assets use
+  an existing [media type](https://www.iana.org/assignments/media-types/media-types.xhtml).
+
+  It is expected to see git repositories be a common directory asset, the
+  `resource_type` to use will be `application/x-git`.
```

Comment:
This reads a little strangely, how about "Git repositories should use the value  'application/x-git'"?